neural_network/02-imdb-binary-classification.ipynb

```diff
@@ -0,0 +1,279 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "C:\\Users\\Hussnain\\Anaconda3\\envs\\tensorflow\\lib\\site-packages\\h5py\\__init__.py:36: FutureWarning: Conversion of the second argument of issubdtype from `float` to `np.floating` is deprecated. In future, it will be treated as `np.float64 == np.dtype(float).type`.\n",
```

Comment:
Should this really be hardcoded to a location on your local disk?

Reply:
Not at all, it's just the output, showing the location of the package and libraries in my drive.

Reply:
When tensorflow starts it shows the location of tensorflow and other packages.